### PR TITLE
feat: Add kas chart

### PR DESCRIPTION
**Pull request by maciaszczykm**
## Checklist

- [ ] If required, I have updated the Plural documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] I have added a meaningful title and summary to convey the impact of this PR to a user.
- [ ] I have added relevant labels to this PR to help with categorization for release notes.

### CODE REVIEW

File: plural/plural/recipe/console-aws.yaml [hunk 1]
```diff
@@ -40,6 +40,9 @@ sections:
   - name: console_dns
     documentation: Fully Qualified Domain Name for the console dashboard, eg console.topleveldomain.com if topleveldomain.com is the hostname you inputed above.
     type: DOMAIN
+  - name: kas_dns
```
Comment (michaeljguarino) [inline plural/plural/recipe/console-aws.yaml:43]: let's not add this yet

Comment (floreks) [reply plural/plural/recipe/console-aws.yaml:43]: Should we only add and push helm part? Will we be able to install and test it?

File: plural/helm/kas/templates/configmap.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    redis:
+      server:
+        address: "{{ printf "%s-master" (include "common.names.fullname" (index .Subcharts "redis")) }}:{{ .Values.redis.master.service.ports.redis }}"
+      key_prefix: {{ include "kas.name" . }}
+      password_file: "/etc/kas/redis_server_secret"
+    api:
+      listen:
+        address: ":{{ .Values.service.apiInternalPort }}"
+        authentication_secret_file: "/etc/kas/.api_secret"
+        certificate_file: /etc/kas/tls.crt
```
Comment (michaeljguarino) [inline plural/helm/kas/templates/configmap.yaml:33]: i'm not totally sure but if the ingress is configured to terminate tls are these necessary?

Comment (floreks) [reply plural/helm/kas/templates/configmap.yaml]: Yep, it was just a leftover. Cleaned up now.

**Review by michaeljguarino — APPROVED** [review]
awesome

**Pull request merged**